PluginSource/source/PlatformBase.h

```diff
@@ -62,12 +62,15 @@
 #elif UNITY_OSX || UNITY_LINUX
 	#define SUPPORT_OPENGL_UNIFIED 1
 	#define SUPPORT_OPENGL_CORE 1
-#elif UNITY_EMBEDDED_LINUX
+#elif UNITY_EMBEDDED_LINUX || UNITY_QNX
 	#define SUPPORT_OPENGL_UNIFIED 1
 	#define SUPPORT_OPENGL_ES 1
 	#ifndef SUPPORT_VULKAN
 		#define SUPPORT_VULKAN 0
 	#endif
+#elif UNITY_QNX
```

Comment:
`|| UNITY_QNX` and `elif UNITY_QNX`. This looks like a mix up?

Reply:
Yes thanks